backend/doc/utils/text_utils.py

```diff
@@ -1,29 +1,69 @@
 import re
+
 from doc.config import settings
 
+
 logger = settings.logger(__name__)
 
 
 def normalize_spaces(text: str) -> str:
     return re.sub(r"\s+", " ", text).strip()
 
 
+_ROMAN_TO_INT = {
+    "I": "1",
+    "II": "2",
+    "III": "3",
+}
+
+
 def normalize_localized_book_name(localized_book_name: str) -> str:
```

Comment:
This is a clever way to be generous in how we receive book names from the field.